### DISCUSSION TITLE
A stupid question. How can I get keyboad hint or use help to get the key I want.

### DISCUSSION THREAD

**Discussion by GuangDai** [LunarVim/LunarVim#2433] [Q&A] [answered]
Before that I have been working inside various IDEs. Now I just use LunarVim, which gives me a new experience. However, since I am new to this tool, I don't know the functions of many keys.

On this point, I checked the documentation and Google and found that which-key seems to solve my need. But even after I checked the documentation, I couldn't configure the which-key inside lunar vim.

In the configuration file, it shows TODO: User Config for predefined plugins. Does this mean I can't configure a more suitable which-key prompt for me now?

Or is there another way to prompt me for what key can do what function?

**Comment by kylo252** [comment, marked as answer]
No such thing as a stupid question!

The way that which-key helps you out is by displaying all possible combination after the first part is triggered, most of the keybinds that are added by lunarvim are prefixed with `<leader>` which should be your `space` key by default. 

tl;dr; press `space` and see a list of most of the custom mappings in lvim, also check `:WhichKey`

**Reply by GuangDai** [reply]
Thanks! I am using it and it's excellent.
